video_search/vstream/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class video_info(models.Model):
```

Comment:
Class name should be camel cased. So VideoInfo works better. But even a better name will be `Videos`.